### DISCUSSION TITLE
New Release?

### DISCUSSION THREAD

**Discussion by Tyharo1** [ivanvotti/ember-svg-jar#229] [Q&A] [answered]
Hey everyone, no rush but wanted to see if there was any plan for a new release? It looks like the last release was in February a lot has gone into the repo since then 😄

**Comment by jherdman** [comment]
Yeah, I think that can be arranged. I'll try to whip something up shortly. I just need to audit the changes first.

**Comment by jherdman** [comment, marked as answer]
Done!
